src/explore-education-statistics-admin/src/pages/admin-dashboard/components/PublicationSummary.tsx

```diff
@@ -34,7 +34,7 @@ const PublicationSummary = ({
     );
 
   // BAU-404 - temporarily hide the Amend Release button completely until Release Versioning Phase 1 is complete
-  const showAmendmentButton = () => false;
+  const showAmendmentButton = () => true;
```

Comment:
Can we please return this to false until the amendment functionality has been tested?

Reply:
done